.gitignore

```diff
@@ -7,6 +7,10 @@ validationTestsJunit.xml
 .bash_history
 *.DS_Store
 
+build_engine_test_images/_pycache__
```

Comment:
why we need these build_engine_test_images specific lines?

Reply:
Removed